src/xdocs/config_design.xml

```diff
@@ -1508,40 +1508,29 @@ class Test {
         </source>
         <p>Example with default values:</p>
         <source>
-public class MyClass {
```

Comment:
I don't see why we are dropping class from these examples.

Reply:
Don't we want to keep it as short as possible?

Reply:
This check is specifically for class members, why keep the class in the example as well? I agree that a class definition should be kept if the check is more general though.

We can also point to the example file in the repo if somebody needs more context.

Reply:
We are going to use `// xdoc section -- start` to mark what section of example code to show in xdoc. It is good idea that @stoyanK7  suggested recently, it simplified (no need to share in final xdoc) a lot of stuff: imports, packages, sibling classes definition .

Reply:
@stoyanK7 , please share links to discord where we discussed this. Or may be we published this in GitHub somewhere

Reply:
Putting class definition back as per https://github.com/checkstyle/checkstyle/pull/13276#discussion_r1237945574. One moment.

Reply:
@rnveach Done.